src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
```

Comment:
- [ ] @sonyco-4u and @Oklukeok, great work adding test for the App component, however it's the project requirement to add tests for other components and also all functionalities in calculate and operate js files, as shown in the screenshot below, kindly add the tests
![image](https://user-images.githubusercontent.com/61408860/201189804-0c5b666c-794c-4695-970b-517361720c3a.png)

Reply:
Corrections done